docs/ui-components/code/line-nos.md

```diff
@@ -1,15 +1,11 @@
 ---
-layout: default
-title: Code with line numbers
+title: Line Numbers
 parent: Code
-grand_parent: UI Components
-permalink: /docs/ui-components/code/line-numbers/
```

Comment:
Just to double-check: the filepath for this file is different from the old `permalink`. Is that intentional? The old permalink is now broken.

Reply:
Good catch. I guess the spelling difference between `line-nos` and `line-numbers` was accidental; I've now renamed the file to `line-numbers`.